Cargo.lock

```diff
@@ -238,6 +238,29 @@ dependencies = [
  "backtrace",
 ]
 
+[[package]]
+name = "apache-avro"
```

Comment:
We now have two different versions of `avro`. 😕

Reply:
I think this one is used in iceberg-rust and it will not expose the interface about apache-avro.

Reply:
We have three, another one in icelake